content/events/2018-detroit/speakers/armon-dadgar.md

```diff
@@ -0,0 +1,10 @@
++++
+Title = "Armon Dadgar"
+Twitter = "armon"
+image = "armon-dadgar.png"
```

Comment:
so this actually works? seems you're providing a full path on the talk page; do we really have to specify images twice? :(

Reply:
correct